### PR TITLE
[ru] testing bypass - `1.0.0` -> `1.0.1`

### PR DESCRIPTION
**Pull request by IgorPolyakov**

### CODE REVIEW

File: lessons/ru/testing/bypass.md [hunk 1]
```diff
@@ -266,6 +269,8 @@ defmodule ClinicTest do
 end
 ```
 
-В этом тесте не должно быть ничего неожиданного. Вместо создания одного `Bypass`-подключения в `setup`, мы создаем два с передачей им портов 1234 и 1337. Дальше — вызовы `Bypass.expect/2` и тот же код, что был у нас в `SchedulerTest` для запуска планировщика и проверки, что соответствующие сообщения записываются в лог.
+В этом тесте не должно быть ничего неожиданного.
+Вместо создания одного `Bypass`-подключения в `setup`, мы создаем два с передачей им портов 1234 и 1337.
```
Comment (nscyclone) [inline lessons/ru/testing/bypass.md:273]:
```suggestion
Вместо создания одного `Bypass`-подключения в `setup` мы создаем два с передачей им портов 1234 и 1337.
```
Feels like an unnecessary comma missed in the earlier version.

**Review by nscyclone — CHANGES_REQUESTED** [review]
Thanks @IgorPolyakov 
There is a suggestion for a minor syntaxic improvement to the previous version of the lesson.

**Comment by IgorPolyakov** [comment]
Thanks for the review, I have made the suggested syntactic improvements)

**Review by nscyclone — APPROVED** [review]
Thanks @IgorPolyakov

**Pull request merged**